nevergrad/functions/images/core.py

```diff
@@ -205,3 +200,49 @@ def _with_tag(
         assert func._initialization_kwargs is not None
         func._initialization_kwargs["tags"] = tags
         return func
+
+
+class ImageFromPGAN(base.ExperimentFunction):
+    """
+    Creates face images using a GAN from pytorch GAN zoo trained on celebAHQ and optimizes the noise vector of the GAN
+
+    problem_name: the type of problem we are working on.
+    initial_noise: the initial noise of the GAN. It should be of dimension (1, 512). If None, it is defined randomly.
+    use_gpu: whether to use gpus to compute the images
+    scorer: which scorer to use for the images
+    mutable_sigma: whether the sigma should be mutable
+    n_mutations: number of mutations
+    """
+
+    def __init__(self, initial_noise: np.ndarray = None, use_gpu: bool = True, scorer=Koncept512Loss, mutable_sigma=True, n_mutations=35) -> None:
```

Comment:
same thing here, better describe it with a string if possible. Unless you plan to have many of those or you want to experiment easily offline with different ones, but in this case we'll need to store some information about them